sql_con.py

```diff
@@ -273,32 +277,56 @@ def update_expiry_date_by_discord_id(self, discord_id, expiry_date):
         return self.dbcon_manager(sql)
     
     def insert_user_into_trade_volume_table(self, uuid, discord_id, date):
-        sql = f"""INSERT INTO {self.config.TRADE_VOLUME_TABLE}
+        sql = f"""
+        INSERT INTO {self.config.TRADE_VOLUME_TABLE}
         (uuid, discord_id, vip_expired_date)
         VALUES
         ('{uuid}', '{discord_id}', '{date}')"""
         return self.dbcon_manager(sql)
     
     def insert_solely_uid(self, uuid):
-        sql = f"""INSERT INTO {self.config.TRADE_VOLUME_TABLE}
+        sql = f"""
+        INSERT INTO {self.config.TRADE_VOLUME_TABLE}
         (uuid)
         VALUES
         ('{uuid}')"""
         return self.dbcon_manager(sql)
 
     def update_user_from_trade_volume_table(self, uuid, discord_id):
-        sql = f"""UPDATE {self.config.TRADE_VOLUME_TABLE}
+        sql = f"""
+        UPDATE {self.config.TRADE_VOLUME_TABLE}
         SET discord_id = '{discord_id}'
         WHERE uuid ='{uuid}'"""
         return self.dbcon_manager(sql)
     
     def update_user_from_trade_volume_table_with_date(self, uuid, discord_id, date):
-        sql = f"""UPDATE {self.config.TRADE_VOLUME_TABLE}
+        sql = f"""
+        UPDATE {self.config.TRADE_VOLUME_TABLE}
         SET discord_id = '{discord_id}', vip_expired_date = '{date}'
         WHERE uuid ='{uuid}'"""
         return self.dbcon_manager(sql)
 
     def fetch_user_trade_volume_by_discord_id(self, discord_id):
-        sql = f"""SELECT volume FROM {self.config.TRADE_VOLUME_TABLE} 
-                  WHERE discord_id='{discord_id}'"""
+        sql = f"""
+        SELECT volume FROM {self.config.TRADE_VOLUME_TABLE} 
+        WHERE discord_id='{discord_id}'"""
         return self.dbcon_manager(sql)
+
+    def fetch_expired_vips(self):
+        sql = f"""
+        SELECT discord_id
+        FROM {self.config.TRADE_VOLUME_TABLE}
+        WHERE DATE(vip_expired_date) = CURDATE()
+        """
+        result = self.dbcon_manager(sql, get_all=True)
+        return [row['discord_id'] for row in result] if result else []
+
+    def fetch_seven_days_expiry_vips(self):
```

Comment:
We can combine this and above function. Since they do the same thing and only differ by number of days, we can make day as a param

Reply:
fixed